integration/apps/wrf/0001-Whitespace-cleanup.patch

```diff
@@ -0,0 +1,1168 @@
+From 4c87740e2e59135c9f9b275e4b1d9c6cda147972 Mon Sep 17 00:00:00 2001
```

Comment:
Why is whitespace clean up required?  Seems like we asking for a future conflict by having this patch that serves little value.  Please remove this patch

Reply:
Whitespace cleanup was added following the examples from hpcg & nekbone
https://github.com/geopm/geopm/blob/dev/integration/apps/hpcg/0001-Fix-whitespace-issues.patch
https://github.com/geopm/geopm/blob/dev/integration/apps/nekbone/0001-Fix-whitespace-issues.patch